Header header
geometry_msgs/Vector3 thrust
geometry_msgs/Vector3 angular_velocity
geometry_msgs/Vector3 angular_velocity_des
geometry_msgs/Vector3 angular_velocity_dot_ref
float64[] rot